src/code.ts

```diff
@@ -0,0 +1,109 @@
+import ApiService from "./apiCall";
+import ebgOptions from "./../formOptions.js";
+
+figma.showUI(__html__, {
+	title: "Erase Bg",
+	height: 400,
+	width: 248,
+	themeColors: true,
+});
+
+figma.ui.onmessage = async (msg) => {
+	if (msg.type === "initialCall") {
+		const body = {
+			type: "createForm",
```

Comment:
same for the createForm ,  savedFormValue , 	if (msg.type === "transform") {
 etc